장운진/week3/Sliding Window Maximum.js

```diff
@@ -0,0 +1,33 @@
+var maxSlidingWindow = function (nums, k) {
+  const answer = [];
+
+  for (let i = 0; i < nums.length; i++) {
+    const currentWindow = nums.slice(i, k + i);
+
+    if (currentWindow.length === k) {
+      answer.push(Math.max(...currentWindow));
+    }
```

Comment:
```suggestion
    answer.push(Math.max(...currentWindow));
```
for 문의 범위를 수정하여 조건문을 없애보는 걸 어떨까요?